examples/market_maker.py

```diff
@@ -0,0 +1,19 @@
+from ucf_exchange_client import Strategy
+
+market_maker = Strategy('mm')
+
+@market_maker.handle('Hello')
+def initialize(state, msg):
+    state.theos = {}
+    return []
+
+@market_maker.handle('Trade')
+def update_values(state, msg):
+    if msg.ticker not in state.theos:
+        state.theos[msg.ticker] = msg.price
+    else:
+        state.theos[msg.ticker] = (state.theos[msg.ticker] + msg.price) / 2
+    return []
+
+
+market_maker.run('localhost', 8166)
```

Comment:
I think I meant this more as a rhetorical question. in any case, you should import-guard this:

```python
if __name__ == "__main__":
  market_maker.run('localhost', 8166)
```